### PR TITLE
Manufacturer Requirements and Process Updates

### PR DESCRIPTION
**Pull request by samgutentag**

### CODE REVIEW

**Comment by samgutentag** [comment]
Absorbing content from PRs:
#1109 - 5G Hotspot Hardware Specifications
#1111 - 5G Hotspot Audit Process
#1112 - Hotspot hardware audit requirements

File: docs/hotspot-makers/5g-hardware-specification.mdx [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+pagination_label: 5G Hardware Specification
+sidebar_label: 5G Hardware Specification
+description: Helium Documentation
+image: https://i.imgur.com/bWAbESE.png
+slug: /hotspot-makers/5g-hardware-specification
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+Hotspots provide sensor devices access to the Helium network via radio. In addition, some Hotspots
```
Comment (deasydoesit) [inline docs/hotspot-makers/5g-hardware-specification.mdx]: > Hotspots provide sensor devices access to the Helium network via radio

Wondering if `sensor devices` and `radio` are the right words in this context?

File: docs/hotspot-makers/5g-hardware-specification.mdx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+---
+
+## Enhanced Secure Element
+
+Helium has always required a Secure Element for all Hotspots that participate in LoRaWAN (soon to be
+IOT) PoC rewards. This requirement will continue with the 5G network, however, the security
+requirements for 5G will be more strict than what is in place for LoRaWAN. To prevent confusion,
+these new requirements are known as **Enhanced Secure Element\***.\*
+
```
Comment (deasydoesit) [inline docs/hotspot-makers/5g-hardware-specification.mdx:84]: `**Enhanced Secure Element\***.\*` rendered as `Enhanced Secure Element*.*` but I'm not sure if that was the intention

File: docs/hotspot-makers/5g-hardware-specification.mdx [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+4. Is the private key encrypted when it is stored in this memory?
+5. If so, where is the key necessary to decrypt the encrypted private key stored?
+6. Again, if the key is encrypted, does each device possess its own unique storage key, or is it
+   shared across all devices?
+7. Again, if the key is encrypted, is there also a verification check to ensure that the key has
+   decrypted properly when it is loaded?
+8. How is the trusted code loaded?
+9. How is the code checked for authenticity when it is loaded?
+10. Who in your organization has access to the keys used for signing this code?
+11. Your implementation should have the ability to be updated. Please share the update method. \*\*
```
Comment (deasydoesit) [inline docs/hotspot-makers/5g-hardware-specification.mdx]: Lingering `\*\*`?

File: docs/hotspot-makers/5g-hardware-specification.mdx [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+### Re-Audit Requirements
+
+Re-audit (such as a change in model) will be required only if any of the following:
+
+- Change from an indoor model to an outdoor model
+- CPU changes
+- RAM **decreases**
+- Storage **decreases**
+- Change in **radio or radio location. \*\***No re-audit is required if one SX130x PCIe card with
+  the same chipset is swapped for another, such as SX1302 for SX1303.\*
```
Comment (deasydoesit) [inline docs/hotspot-makers/5g-hardware-specification.mdx]: This list item looked off

File: docs/hotspot-makers/5g-hardware-specification.mdx (hunk 1 above)
Comment (samgutentag) [reply docs/hotspot-makers/5g-hardware-specification.mdx]: going to pull this entire paragraph. It is not 5G hardware specific and the content is covered well in other documents

File: docs/hotspot-makers/5g-hardware-specification.mdx (hunk 2 above)
Comment (samgutentag) [reply docs/hotspot-makers/5g-hardware-specification.mdx:84]: good catch, notion to markdown copy paste artifact

File: docs/hotspot-makers/5g-hardware-specification.mdx (hunk 4 above)
Comment (samgutentag) [reply docs/hotspot-makers/5g-hardware-specification.mdx]: formatting artifact, fixed!

**Review by jthiller — APPROVED** [review]

**Pull request merged**